db/migrate/20181002145749_add_workflow_steps.rb

```diff
@@ -0,0 +1,5 @@
+class AddWorkflowSteps < ActiveRecord::Migration
+  def change
+    add_column :workflows, :steps, :jsonb
```

Comment:
I think this should have a default, to avoid serializing null. From braintree: https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/
```
Add column, add default as a separate command, and backfill the column with the default value
```

Reply:
#2933 specifically requested that it default to NULL. @srallen what was the reason for that?

Reply:
I think personally I'd rather also add a classifier_version attribute.

Reply:
Ok. That's alright with me. The default value for the `classifier_version` could be `1.0` and if `steps` are defined, it is then changed to `2.0`.